HumanDynamicsEstimationLibrary/algorithms/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+find_package(OsqpEigen 0.4.0 REQUIRED)
+find_package(YARP 3.2 REQUIRED)
+find_package(iDynTree REQUIRED)
+
+# Include AddComponent cmake module
+include(AddComponent)
```

Comment:
If you use it for all targets, maybe you could include the macro in the top-level CMakeLists